resources/startup.sh

```diff
@@ -112,131 +103,104 @@ function writeProxyCredentialsTo(){
 
 function writeProxyAuthenticationCredentialsTo(){
   # Check for java option and add it if not existent
-  if [ -z "$(grep sonar.web.javaAdditionalOpts= "$1" | grep Djdk.http.auth.tunneling.disabledSchemes=)" ]; then
+  if [[ -z "$(grep sonar.web.javaAdditionalOpts= "$1" | grep Djdk.http.auth.tunneling.disabledSchemes=)" ]]; then
     sed -i '/^sonar.web.javaAdditionalOpts=/ s/$/ -Djdk.http.auth.tunneling.disabledSchemes=/' "$1"
   fi
   # Add proxy authentication credentials
-  echo http.proxyUser="${PROXYUSER}" >> ${SONAR_PROPERTIESFILE}
-  echo http.proxyPassword="${PROXYPASSWORD}" >> ${SONAR_PROPERTIESFILE}
+  echo http.proxyUser="${PROXYUSER}" >> ${SONAR_PROPERTIES_FILE}
+  echo http.proxyPassword="${PROXYPASSWORD}" >> ${SONAR_PROPERTIES_FILE}
+}
+
+function set_property_with_default_admin_credentials() {
```

Comment:
All the curl commands should fail if we get a response like 401

Reply:
Added the --fail parameter, they should fail on unsuccessful response codes now.